### DISCUSSION TITLE
New Contributor

### DISCUSSION THREAD

**Discussion by akash-electron** [ruxailab/RUXAILAB#538] [Q&A]
Hello, I am Akash Pandey, computer engineer undergraduate student . I would love to contribute to open source projects of your organization.
 Could you please let me know how to start with it?

**Comment by marcgc21** [comment]
Hi @akash-electron , we are closing the creation of threads inside Discussions because it is hard to maintain for the organization. We prefer to work on Discord channels. There you will find the best place to know how to contribute to the organization, the repositories and also GSOC or other calls that we are running.
